.github/workflows/ci.yml

```diff
@@ -153,6 +156,9 @@ jobs:
                      -DCMAKE_PREFIX_PATH=${GITHUB_WORKSPACE}/install/deps \
                      -DCMAKE_BUILD_TYPE=${{ matrix.build_type }} \
                      -DCMAKE_INSTALL_PREFIX=${GITHUB_WORKSPACE}/install/deps \
+                     -DINCLUDE_PREFIX:PATH="include" -DCMAKE_PREFIX:PATH="lib/cmake/casadi" \
+                     -DLIB_PREFIX:PATH="" \
```

Comment:
why `""` and not `"lib"`?

Reply:
ops

Reply:
and actually also why `""` and not `bin`?